doc-build/md001.md

```diff
@@ -26,6 +26,15 @@ level at a time:
 ### Another Heading 3
 ```
 
+If [YAML](https://en.wikipedia.org/wiki/YAML) front matter is present and
+contains a `title` property (commonly used with blog posts), this rule treats
+that as a top level heading and will report a violation if the actual first
+heading is not a level 2 heading.
+To use a different property name in the front matter,
```

Comment:
Is this paragraph copied from a different rule? The spacing is weird here - should this line be part of the previous?

Reply:
Yes, copied from rule 25 doc. I updated the line endings. If this still does not comply with what you'd expect, can you please suggest the exact formatting you'd like?

Reply:
In MD025, there's no break between the two sentences: https://github.com/DavidAnson/markdownlint/blob/224987d727ebada36752074d2466a061c9e03ff0/doc-build/md025.md?plain=1#L29

That's the convention.